spec/features/user_profile_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe 'user show page' do
+  let!(:location) { create :location }
+  let!(:user) { create :active_user, location: location }
+  let!(:other_user) { create :active_user, location: location }
+
+  before do
+    login_as(user, scope: :user)
+  end
+
+  it 'should be viewable by other users' do
+    visit user_path(other_user)
+    expect(page).to have_content other_user.full_name
+  end
+
+  it 'should show other users in peer group' do
+    PeerGroup.generate_groups
+    visit user_path(user)
+    expect(page).to have_content other_user.full_name
+  end
+end
```

Comment:
Could you add a few more tests such as: It does not show users that are not your peers. It does not show past peers (only the current group).

What should we do about peers when they are the last peer group, but maybe you haven't been participating the last few months, so the peer group is old?